files/pull-image.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
copied from `pull-sandbox-image.sh` but made generic to use in install-worker for caching other images.

Reply:
this file is indented with tabs, can you convert to spaces? (we need to add some linting, note to self)